streampark-common/pom.xml

```diff
@@ -128,6 +128,32 @@
             <version>${streampark.shaded.version}</version>
         </dependency>
 
+        <!-- ZIO -->
+        <dependency>
```

Comment:
"common" is a public module that many other modules depend on. It is not a good idea to add ZIO dependency in the common module.

Reply:
The interfaces exposed by streampark-flink-kubernetes are all zio or zstream structures, and external calls must refer to zio. I'll be writing more zio code in other modules later on.ZIO is a basic, pure-scala, zero-dependency library, so placing it in comon doesn't seem like a problem.